project/Dependencies.scala

```diff
@@ -15,6 +15,7 @@ object Dependencies {
     val scalaTest = "2.1.0-RC2"
     val simpleArm = "1.1.10"
     val slf4j = "1.7.5"
+    val socrataHttpServer = "3.1.0"
```

Comment:
You don't need this.  socrataHttpCuratorBroker already pulls in 3.3.0, and you are pulling in a lower version.